exp-fashion-mnist/fashionmnist-train.py

```diff
@@ -103,51 +94,21 @@ def _prep():
     client.run(_prep)
 
     def submit(seed, **kwargs):
-        # import train
-
-        # assert train.__version__ == "0.1"
-
-        # import adadamp
-        #
-        # assert adadamp.__version__ == "0.2.0rc4"
-
         return train.main(
             epochs=epochs,
             verbose=True,
-            init_seed=seed,
-            random_state=seed,
-            tuning=True,
+            tuning=False,
             **kwargs,
         )
 
     futures = []
     seeds = np.arange(seed_start, seed_start + n_runs)
-    dampers = ["geodamp", "adagrad", "geodamplr", "radadamp"]
-    #  assert set(dampers).issubset(set(params.keys()))
-
-    param_dist = {
-        "lr": [0.005],
-        "momentum": [0.9],
-        "nesterov": [True],
-        "rho": [0.95, 0.99, 0.995, 0.999],
-        "dampingfactor": np.logspace(0, 1, num=1000),
-        "dampingdelay": [2, 5, 10, 20, 30, 60],
-        "initial_batch_size": [16, 32, 64, 128, 256],
-        "max_batch_size": [256, 512, 1024, 4096],
-        "weight_decay": [3e-3, 1e-3, 3e-4, 1e-4],
-        "dwell": [1, 10, 100, 300, 1000, 3000],
-    }
-    n_iters = {
-        "radadamp": 30,
-        "geodamp": 30,
-        "adagrad": 6,
-    }
-    for damper in dampers:
-        if damper not in n_iters:
-            continue
-        damper_params = ParameterSampler(param_dist, n_iter=n_iters[damper])
-        for params in damper_params:
-            futures.extend(client.map(submit, seeds, damper=damper, **params))
+
+     with open("hyperparams.json", "r") as f:
+         paramslt = json.load(f)
```

Comment:
I think we should run 

```suggestion
     paramslt["geodamplr"] = deepcopy(paramslt["geodamp"])
     paramslt["geodamplr"]["max_batch_size"] = paramslt["geodamplr"]["initial_batch_size"]
     paramslt["radadamplr"] = deepcopy(paramslt["radadamp"])
     paramslt["radadamplr"]["max_batch_size"] = paramslt["radadamplr"]["initial_batch_size"]
     paramslt["radadamplr"]["damper"] = paramslt["radadamplr"]["radadamp"]
```

Regulard SGD (aka `geodamplr`) should definitely be run. I'm also curious to see how RadaDamp with an adaptive learning rate performs.